views/users/view.hbs

```diff
@@ -8,32 +8,181 @@
     <div class="col-sm-9">
 
       <h1>
-        User profile: {{person.name}}
+        User profile: {{person.orcidUserData.userFullName}}
       </h1>
 
       <br />
 
       <strong>ORCiD:</strong>
-      <a href="https://orcid.org/{{person.orcid}}" target="_blank">
-        {{person.orcid}}
+      <a href="https://orcid.org/{{person.orcidId}}" target="_blank">
+        {{person.orcidId}}
         <span title="This link opens in a new tab." class="fa fa-external-link"></span>
       </a>
 
-      <!-- <pre>{{json user}}</pre> -->
+      <br /> <br />
+
+      <strong>Current and past affiliations:</strong>
+      {{#if person.orcidUserData.userEmployments.currentEmployment}}
+      <ul class="list">
+        <li>
+          <b>{{person.orcidUserData.userEmployments.currentEmployment.organizationName}}</b> -
+          {{person.orcidUserData.userEmployments.currentEmployment.roleTitle}}
+        </li>
+
+        {{#if person.orcidUserData.userEmployments.pastEmployments}}
+        <div id="pastAffiliations" style="display:none">
+          {{#each person.orcidUserData.userEmployments.pastEmployments}}
+          <li>
+            <b>{{this.organizationName}}</b> - {{this.roleTitle}}
+          </li>
+          {{/each}}
+        </div>
+
+        <input id="displayAffiliations" type="button" name="expandAffiliations" value="Show past affiliations"
+          onclick="showPastAffiliations()" />
+        {{/if}}
+      </ul>
+      {{/if}}
+
+      <br />
+
+      <strong>Educational background:</strong>
+      <ul class="list">
+        {{#each person.orcidUserData.userEducations}}
+        <li>
+          <b>{{this.organizationName}}</b> - {{this.roleTitle}}
+        </li>
+        {{/each}}
+      </ul>
+
+      <br />
+
     </div>
 
-    <div class="col-sm-3" id="sidebar">
-      <h3>Publications</h3>
+    <div class="col-sm-3">
+      {{#if publications.length}}
+      <canvas id="myChart" width="250px" height="250px"></canvas>
+      {{/if}}
+    </div>
+  </div>
+
+  {{#if publications.length}}
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Publications</h1>
+
+      <hr>
+
       <ul class="list list-unstyled">
-      {{#each publications}}
-        <li><a href="/publications/view/{{this._id}}">{{this.title}}</a></li>
-      {{/each}}
+        {{#each publications}}
+        <li class="publication-container" onclick="window.location='/publications/view/{{this._id}}'">
```

Comment:
Can we do a simple `<a href="/publications/view/{{this._id}}">` instead, wrapped around `{{this.title}}`?

Doing it the anchor tag way will make it compatible with any browser or crawler.
Also it'll be more accessible (otherwise screen readers may not read this out as a link; plus keyboard navigation suffers with the current implementation).